vyxal/elements.py

```diff
@@ -3207,7 +3207,10 @@ def sort_by(lhs, rhs, ctx):
         function, vector = (
             (lhs, rhs) if ts[0] is types.FunctionType else (rhs, lhs)
         )
-        return sorted(vector, key=lambda x: safe_apply(function, x, ctx=ctx))
+        return sorted(
+            iterable(vector, ctx=ctx),
```

Comment:
This looks like it does digits. Shouldn't it be a range for consistency with `'ƛ`?

Reply:
I think digits is probably more useful for sorting